extras/lbf-nix/lbf-plutus-typescript.nix

```diff
@@ -0,0 +1,39 @@
+# Build .lbf schemas that use LB Plutus (and by extension LB Prelude) package and targets Typescript's plutus-ledger-api library.
+{ pkgs
+, lbf
+, lbg-typescript
+, config
+, typescriptFlake
+}:
+lbfTypescriptOpts:
+let
+  utils = import ./utils.nix pkgs;
+
+  lbfTypescript = import ./lbf-typescript.nix { inherit pkgs lbf lbg-typescript typescriptFlake; };
+  lbfTypescriptOptsForPlutus = utils.overrideAttrs
+    {
+      imports = {
+        default = { };
+        override = libs: libs // {
+          lbf-prelude = ../../libs/lbf-prelude;
+          lbf-plutus = ../../libs/lbf-plutus;
```

Comment:
Use `config.packages.lbf-prelude|plutus` rather than direct paths

Reply:
I see, thank you! This makes much more sense!